tests/testthat/test-qenv_constructor.R

```diff
@@ -4,6 +4,8 @@ testthat::test_that("constructor returns qenv if nothing is specified", {
   testthat::expect_identical(ls(q@env), character(0))
   testthat::expect_identical(q@code, expression())
   testthat::expect_identical(q@id, integer(0))
+  testthat::expect_identical(q@warnings, character(0))
+  testthat::expect_identical(q@messages, character(0))
```

Comment:
We should probably test that if you create a qenv using code = "..." i.e. line 11 then you can also have the correct number of elements in q@warnings (i.e. testing the rep("", length(code)) line in the constructor